pkg/console/subresource/authentication/cluster.go

```diff
@@ -0,0 +1,45 @@
+package authentication
+
+import (
+	"fmt"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/console-operator/pkg/api"
+	"golang.org/x/exp/slices"
+)
+
+func GetOIDCOCLoginCommand(authConfig *configv1.Authentication, apiServerURL string) string {
+	provider, clientConfig := GetOIDCClientConfig(authConfig, api.TargetNamespace, api.CLIOIDCClientComponentName)
+	if provider == nil || clientConfig == nil || provider.Issuer.URL == "" {
+		return ""
+	}
+
+	extraScopes := ""
+	if len(clientConfig.ExtraScopes) > 0 {
+		extraScopes = fmt.Sprintf(" --extra-scopes %s", strings.Join(clientConfig.ExtraScopes, ","))
+	}
+
+	return fmt.Sprintf("oc login %s --issuer-url %s --exec-plugin oc-oidc --client-id %s%s", apiServerURL, provider.Issuer.URL, clientConfig.ClientID, extraScopes)
```

Comment:
This was discussed at first, but decided against as a more complex solution that wasn't really necessary for the current need. At this point, it's probably too late in the game to change as it would require pretty significant changes here and in console.

Reply:
I agree, lets add it as a followUp